apps/api/src/coscrad-cli/ingest-media-items.cli-command.ts

```diff
@@ -288,30 +301,42 @@ export class IngestMediaItemsCliCommand extends CliCommandRunner {
                 throw topLevelError;
             }
 
-            const publicationResult = await this.commandHandlerService.execute(
-                {
-                    type: 'PUBLISH_RESOURCE',
-                    payload: {
-                        aggregateCompositeIdentifier: fsa.payload.aggregateCompositeIdentifier,
-                    },
-                },
-                {
-                    userId: 'COSCRAD Admin',
-                }
+            this.logger.log(
+                `Media Item Added: ${JSON.stringify(fsa.payload.aggregateCompositeIdentifier.id)}`
             );
 
-            if (isInternalError(publicationResult)) {
-                const error = new InternalError(
-                    `Failed to import media item: ${fsa.payload.aggregateCompositeIdentifier.id} at the publication stage`,
-                    [publicationResult]
+            // publish media item, if this was requested
+            if (shouldPublishMediaItems) {
+                const publicationResult = await this.commandHandlerService.execute(
+                    {
+                        type: 'PUBLISH_RESOURCE',
+                        payload: {
+                            aggregateCompositeIdentifier: fsa.payload.aggregateCompositeIdentifier,
+                        },
+                    },
+                    {
+                        userId: 'COSCRAD Admin',
+                    }
                 );
 
-                this.logger.log(error.toString());
+                if (isInternalError(publicationResult)) {
+                    const error = new InternalError(
+                        `Failed to import media item: ${fsa.payload.aggregateCompositeIdentifier.id} at the publication stage`,
+                        [publicationResult]
+                    );
 
-                throw error;
+                    this.logger.log(error.toString());
+
+                    throw error;
+                }
             }
 
-            this.logger.log(`Media Item Added: ${JSON.stringify(fsa)}`);
+            // tag media item, if this was requested
+            if (tags.length > 0) {
```

Comment:
Can we do this now?

Reply:
Let's hold off as this is pretty urgent. We can always tag the recent items separately for now.